packages/component-library/stories/HorizontalBarChart.story.js

```diff
@@ -225,4 +225,50 @@ export default () =>
         );
       },
       { notes }
+    )
+    .add(
+      "Example: Stacked bar chart",
+      () => {
+        const sampleStackedData = [
+          [{ x: "2017", y: 7 }, { x: "2018", y: 2 }, { x: "2019", y: 3 }],
+          [{ x: "2017", y: 2 }, { x: "2018", y: 5 }, { x: "2019", y: 4 }],
+          [{ x: "2017", y: 1 }, { x: "2018", y: 2 }, { x: "2019", y: 3 }]
+        ];
+
+        const dataValue = text("Data value", "x", GROUP_IDS.DATA);
+        const dataValueFormatter = getKeyNames(civicFormat);
+        const optionSelectX = options(
+          "X-axis value format",
+          dataValueFormatter,
+          "percentage",
+          { display: "select" },
+          GROUP_IDS.LABELS
+        );
+        const minimalist = boolean("Minimalist", false, GROUP_IDS.CUSTOM);
+        const dataLabel = text("Data label", "y", GROUP_IDS.DATA);
+        const xLabel = text("X-axis label", "Percent", GROUP_IDS.LABELS);
+        const yLabel = text("Y-axis label", "Year", GROUP_IDS.LABELS);
+        const title = text("Title", "Numbers", GROUP_IDS.LABELS);
+        const subtitle = text(
+          "Subtitle",
+          "that mean nothing",
+          GROUP_IDS.LABELS
+        );
+        const data = object("Data", sampleStackedData, GROUP_IDS.DATA);
+        return (
+          <HorizontalBarChart
+            data={data}
+            dataValueFormatter={x => civicFormat[optionSelectX](x)}
+            dataValue={dataValue}
+            dataLabel={dataLabel}
+            xLabel={xLabel}
+            yLabel={yLabel}
+            title={title}
+            subtitle={subtitle}
+            minimalist={minimalist}
+            stacked
```

Comment:
I tried making 'stacked' a Knob and the chart wouldn't render anything anymore (I assume because the data format also needs to change). Generally, I feel like a single prop shouldn't be dependent on other changes to work. Would it make sense to either make a separate StackedBarChart component, or to make sure a non-stacked array of arrays renders as a grouped bar chart (https://formidable.com/open-source/victory/docs/victory-group/)?

Reply:
I lean towards creating a new component because it requires an array of arrays of objects, rather than an array of objects. I think it could be misleading if we rendered a different type of chart based on the data structure, even if the user passes the `stacked` prop. I am open to other suggestions though!

Reply:
I suggest looking at the Stacked Area Chart and comparing it to the Bar Chart / Stacked Bar Chart.  If users, such as Data Scientists, think about how these two types of charts present data in a similar way (and from my research there is a lot of similarity), it would be good to design the stories, and the data structure for component, the same way.  It will improve usability.

For example, the Stacked Area Chart doesn't have a knob to say "stacked" or not, it's determined by the data file.  Compare the knobs in the Data tabs for the "Standard" and "Example: Simple" stories.  Should this change to be similar to the Bar Chart / Stacked Bar Chart, or should the Bar Chart / Stacked Bar Chart be similar to the Stacked Area Chart?

Reply:
@bradfordjanice Thanks for the advice, I will be sure to checkout that component and use my best judgement on how to proceed.